flasgger/utils.py

```diff
@@ -288,8 +288,9 @@ def __replace_ref(schema, relative_path):
 
 
 def validate(
-        data=None, schema_id=None, filepath=None, root=None, definition=None,
-        specs=None, validation_function=None, validation_error_handler=None):
+        data=None, require_data=True, schema_id=None, filepath=None,
```

Comment:
Maybe it's best to put the new keyword argument at the end so that current uses passing positional arguments are not affected

Reply:
Thanks for the review @billyrrr, that makes sense. I've amended the argument list and tests now pass.